### ISSUE TITLE
xournalpp/nextcloud

### ISSUE THREAD

**Issue by pdupre0** [xournalpp/xournalpp#4783]
- OS: [Linux (fedora) fc36.x86_64
 - (Linux only) Desktop environment: [X11]
 - Which version of libgtk do you use
Xournal++ 1.1.3
└──libgtk: 3.24.35

I use nextcloud on 2 different machines (in addition to the server).
I use xournalpp to edit .pdf files.
Everytime that I have edited the pdf file on one machine, I have an
issue when I want to edit the "same" file on the other machine.
The .pdf file is not "found", I have to re-associate it to the .xopp file
(select another file).
Can I avoid the problem?

**To Reproduce**
1. have 2 machines synchronized with nextcloud.
2. On machine 1, Edit a pdf file, save the xopp
3. On machine 2, Edit the "same file .xopp with xournalpp
An error appears because the pdf file is not found

![xournalpp](https://user-images.githubusercontent.com/66337763/230080988-5b88fdee-8267-4db3-b06c-a19b45f59125.png)

**Comment by rolandlo** [comment]
I do not see your bottom-left, but here it is in my version:

![grafik](https://user-images.githubusercontent.com/40485433/230101335-4375dc00-935a-406c-ab4b-0e39246e2349.png)

**Comment by rolandlo** [comment]
Yes, in that case the PDF is duplicated, which may be an unwanted side effect.
The problem with using the same xopp-file and PDF background from two different machines is most likely that the absolute file paths for the PDF are different.
